src/test/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+akka.actor {
+  serializers {
+    borer-cbor = "example.BorerAkkaSerializer"
```

Comment:
Whooops the tests are not passing now, pls set up CircleCI for this project (https://app.circleci.com/pipelines/github/VirtusLab/akka-safer-serializer)

Reply:
See https://github.com/VirtusLab/infrastructure-as-types/blob/master/.circleci/config.yml for an example config for sbt